src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.Icu.cs

```diff
@@ -420,15 +420,30 @@ private static CultureInfo[] IcuEnumCultures(CultureTypes types)
                 return Array.Empty<CultureInfo>();
             }
 
-            int bufferLength = Interop.Globalization.GetLocales(null, 0);
+            int bufferLength;
+#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS
+            if (GlobalizationMode.Hybrid)
+                bufferLength = Interop.Globalization.GetLocalesNative(null, 0);
+            else
+                bufferLength = Interop.Globalization.GetLocales(null, 0);
```

Comment:
nit: Could maintain file consistency with having braces for the single line if/else statements